features/bevragen/autorisatie/autorisatie.feature

```diff
@@ -357,20 +265,19 @@ Functionaliteit: autorisatie voor het gebruik van de API
       | beschrijving document (82.30)   | D27894-2004-A782 |
       | ingangsdatum geldigheid (85.10) | 20031107         |
       | datum van opneming (86.10)      | 20040112         |
-      Als gba personen wordt gezocht met de volgende parameters
+      Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                                          |
       | type                | RaadpleegMetBurgerservicenummer                 |
       | burgerservicenummer | 000000061                                       |
       | fields              | kinderen.naam.voornamen,kinderen.geboorte.datum |
       Dan heeft de response een persoon zonder 'kind' gegevens
 
 
-  Rule: Een gemeente als afnemer is geautoriseerd voor alle zoekvragen voor haar eigen inwoners
-    Wanneer de afnemer parameter gemeenteVanInschrijving gebruikt 
-    en die is gelijk aan de waarde van gemeenteCode in de 'claim', 
+  Rule: Een gemeente als afnemer is geautoriseerd voor alle zoekvragen
+    Wanneer gemeenteCode in de 'claim' in de OAuth token gevuld is,
     dan wordt niet gekeken naar de autorisatie van de afnemer
 
-    Scenario: Gemeente is niet geautoriseerd voor de zoekparameters maar zoekt alleen eigen inwoners met parameter gemeenteVanInschrijving
+    Abstract Scenario: Gemeente is niet geautoriseerd voor de gebruikte zoekparameter
```

Comment:
welke zoekparameter wordt hier bedoeld?

Reply:
hoe bedoel je? In dit geval is de afnemer voor geen enkele zoekparameter geautoriseerd.

Reply:
en er is in dit geval maar 1 echte zoekparameter gebruikt